prime-router/docs/design/proposals/translation-template-test-plan.md

```diff
@@ -0,0 +1,57 @@
+# Translation Template Test Plan
+
+## Problem Statement
+As a Reportstream engineer, I want tests that verify that the changes I make to translations will not break essential 
+rules set forth by the spec or our users. 
+
+Currently, we use integration tests for this purpose, but these do not give us the ability to easily tell which thing 
+broke/changed or even really what is specifically being tested.
+
+## Considerations
+- We not only want to check the rules set forth in the spec, but the specific ones created for senders and receivers.
+
+## Proposal on *what* to test
+We should write tests when:
+- The spec specifies a condition.
```

Comment:
100% agree, but I think it might be helpful to have some examples of this and to break this down between conditions that exist in the HL7 spec and conditions that exist in the FHIR inventory